package-lock.json

```diff
@@ -4285,6 +4295,16 @@
       ],
       "license": "MIT"
     },
+    "node_modules/fake-indexeddb": {
```

Comment:
package.json is unchanged from what I see. I wonder where the dependency comes from.

Reply:
reporting/package.json lists it as devDependency